### PR TITLE
doc: Fixes Public -> Private Preview and adds contact link.

### PR DESCRIPTION
**Pull request by EspenAlbert**
## Description

Fixes Public -> Private Preview and adds contact link.

Link to any related issue(s): CLOUDP-277628

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [ ] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code
- [ ] If changes include deprecations or removals I have added appropriate changelog entries.
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

**Review by marcosuma — APPROVED** [review]

**Review by kyuan-mongodb — APPROVED** [review]
LGTM!

File: docs/data-sources/resource_policies.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 `mongodbatlas_resource_policies` returns all resource policies in an organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline docs/data-sources/resource_policies.md]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: docs/data-sources/resource_policy.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 `mongodbatlas_resource_policy` describes a resource policy in an organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline docs/data-sources/resource_policy.md]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: docs/resources/resource_policy.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 `mongodbatlas_resource_policy` provides a Resource Policy resource. The resource lets you create, edit and delete resource policies to prevent misconfigurations and reduce the need for corrective interventions in your organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline docs/resources/resource_policy.md]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: examples/mongodbatlas_resource_policy/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # MongoDB Atlas Provider -- Atlas Resource Policy
 This example creates three different resource policies in an organization.
 
-**NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+**NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline examples/mongodbatlas_resource_policy/README.md]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: templates/data-sources/resource_policies.md.tmpl [hunk 5]
```diff
@@ -2,8 +2,8 @@
 
 `{{.Name}}` returns all resource policies in an organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline templates/data-sources/resource_policies.md.tmpl]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: templates/data-sources/resource_policy.md.tmpl [hunk 6]
```diff
@@ -2,8 +2,8 @@
 
 `{{.Name}}` describes a resource policy in an organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline templates/data-sources/resource_policy.md.tmpl]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

File: templates/resources/resource_policy.md.tmpl [hunk 7]
```diff
@@ -2,8 +2,8 @@
 
 `{{.Name}}` provides a Resource Policy resource. The resource lets you create, edit and delete resource policies to prevent misconfigurations and reduce the need for corrective interventions in your organization.
 
--> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
-1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact MongoDB Support).
+-> **NOTE**: Resource Policies are currently in Private Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Resource Policies` Beta Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```
Comment (Zuhairahmed) [inline templates/resources/resource_policy.md.tmpl]:
```suggestion
1. Enable the `Atlas Resource Policies` Preview Feature in your organization (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

**Review by Zuhairahmed — COMMENTED** [review]
we have since moved away from using "beta". feel free to merge after updating.

**Review by AgustinBettati — APPROVED** [review]
LGTM

File: docs/data-sources/resource_policies.md (hunk 1 above)
Comment (EspenAlbert) [reply docs/data-sources/resource_policies.md]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: docs/data-sources/resource_policy.md (hunk 2 above)
Comment (EspenAlbert) [reply docs/data-sources/resource_policy.md]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: docs/resources/resource_policy.md (hunk 3 above)
Comment (EspenAlbert) [reply docs/resources/resource_policy.md]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: examples/mongodbatlas_resource_policy/README.md (hunk 4 above)
Comment (EspenAlbert) [reply examples/mongodbatlas_resource_policy/README.md]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: templates/data-sources/resource_policy.md.tmpl (hunk 6 above)
Comment (EspenAlbert) [reply templates/data-sources/resource_policy.md.tmpl]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: templates/resources/resource_policy.md.tmpl (hunk 7 above)
Comment (EspenAlbert) [reply templates/resources/resource_policy.md.tmpl]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

File: templates/data-sources/resource_policies.md.tmpl (hunk 5 above)
Comment (EspenAlbert) [reply templates/data-sources/resource_policies.md.tmpl]: Done in 560fa58e77edeb8ce399446ea924c549b0422452

**Pull request merged**